### PR TITLE
UPDATE: adding vue js and various linters packages

### PR DESCRIPTION
**Pull request by dannewnsjump24**
This brings in vue js, updates the linters and standards used from the frontend. Its worth just checking through to make sure this looks 100% ok to everyone.

### CODE REVIEW

File: resources/js/bootstrap.js [hunk 1]
```diff
@@ -48,3 +49,10 @@ orderFilters.init();
 
 const adminMenu = new AdminMenu();
 adminMenu.init();
+
+Vue.config.devtools = true;
```
Comment (AlexJump24) [inline resources/js/bootstrap.js:53]: Should we set this to false now and also turn silent mode on?

File: resources/js/float-label.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import FloatLabels from 'float-labels.js';
 
-var floatlabels = new FloatLabels('form', {
+const floatlabels = new FloatLabels('form', {
```
Comment (annekam) [inline resources/js/float-label.js:3]: I'm not sure if this is actually being used at the moment

File: resources/js/bootstrap.js (hunk 1 above)
Comment (dannewnsjump24) [reply resources/js/bootstrap.js:53]: Yes I will change that to false and put the silent mode on

File: resources/js/float-label.js (hunk 2 above)
Comment (dannewnsjump24) [reply resources/js/float-label.js]: Ive now removed this

**Pull request merged**